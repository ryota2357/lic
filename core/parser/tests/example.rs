//! Generated by "../test_data/gen.ts". Do not edit by hand.
#![cfg_attr(rustfmt, rustfmt_skip)]
mod macros;
test!(fibonacci_z_combinator, "../../../tests/cases/fibonacci_z_combinator/main.lico");
test!(fizz_buzz_for, "../../../tests/cases/fizz_buzz_for/main.lico");
test!(fizz_buzz_while, "../../../tests/cases/fizz_buzz_while/main.lico");
test!(hello_with_variable, "../../../tests/cases/hello_with_variable/main.lico");
test!(hello_world, "../../../tests/cases/hello_world/main.lico");
