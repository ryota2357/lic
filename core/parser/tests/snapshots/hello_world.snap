---
source: parser/tests/example.rs
description: hello_world
---
PROGRAM@0..25
  EXPR_STMT@0..25
    CALL_EXPR@0..24
      LOCAL_VAR@0..7
        IDENT@0..7 "println"
      ARG_LIST@7..24
        OPENPAREN@7..8 "("
        LITERAL@8..23
          STRING@8..23 "\"Hello, world!\""
        CLOSEPAREN@23..24 ")"
    WHITESPACE@24..25 "\n"
